src/main/java/space/obminyashka/items_exchange/util/JwtUtil.java

```diff
@@ -0,0 +1,33 @@
+package space.obminyashka.items_exchange.util;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.concurrent.TimeUnit;
+
+@Component
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class JwtUtil {
+
+    public static final String DATE_FORMAT = "yyyy-MM-dd HH:mm:ss";
+
+    @Value("${app.access.jwt.expiration.time.ms}")
+    private static long jwtAccessTokenExpirationMillis;
```

Comment:
Right now JwtTokenProvider has that value as well. It would be much better to have one storage of that value and to move this methods into provider or to cut provider's functionality and to move all related logic into current class.
How to make it - it's up to you.

Reply:
Ok